templates/partner/index.html.twig

```diff
@@ -13,7 +13,11 @@
                         <h3 class="partner_title card-title">{{ partner.name }}</h3>
                     </div>
                     <div class="partner_card_image">
-                        <a class="partner_link_image"><img class="partner_image card-img-top" src="{{ asset( partner.poster ) }}" alt="Card image cap"  title="Cliquer sur l'image pour en savoir plus !" /></a>
+                    {% if partner.poster.slug is defined %}
```

Comment:
Are you sure that it couldn't be defined ?
You probably should test if it's not empty instead

Reply:
Ok 'is not empty' would be more effective...